app/client/components/transferAccount/StandardTransferAccountList.tsx

```diff
@@ -311,7 +311,8 @@ class StandardTransferAccountList extends React.Component<
 
     const isViewer = !(
       this.props.adminTier === "superadmin" ||
-      this.props.adminTier === "sempoadmin"
+      this.props.adminTier === "sempoadmin" ||
+      this.props.adminTier === "admin"
```

Comment:
@michielderoos please review + verify this fixes the issue

Reply:
This looks good to me!